.github/PENDING_REVIEW_NOTIFICATION_TEMPLATE.md

```diff
@@ -0,0 +1,8 @@
+Hi {{ username }},
+
+It looks like you haven't reviewed the following Oppia-web PRs within the expected time:
+{{ pr_list }}
+
+Can you please review the pending PRs as soon as possible?
```

Comment:
Per my comment on the oppia codeowner discussions thread, would it be possible to change this line to:

"Could you please review the pending PRs as soon as possible by going to [this link](https://github.com/pulls/assigned)?" 

That way, it's easy to see all the PRs in one place. Thanks!

Reply:
Done!